database/tables/group_configs.pg

```diff
@@ -0,0 +1,20 @@
+columns
+    assessment_id: bigint
+    course_instance_id: bigint not null
+    date: timestamp with time zone default CURRENT_TIMESTAMP
+    deleted_at: timestamp with time zone
+    id: bigint not null default nextval('group_configs_id_seq'::regclass)
+    maximum: integer
+    minimum: integer
+    name: text
+    type: text
```

Comment:
If I'm understanding correctly, this `type` is storing an integer flag? Instead, let's store flags or strings to indicate specific capabilities. For example, if we are turning on the ability for students to self-schedule, let's have a boolean flag for that.

Reply:
will change to different booleans {student_auth_join, student_auth_create, student_auth_quit} inside the group_config table